Loop/Managers/DeviceDataManager.swift

```diff
@@ -859,6 +936,7 @@ class DeviceDataManager: CarbStoreDelegate, TransmitterDelegate {
 
         defer {
             transmitterID = NSUserDefaults.standardUserDefaults().transmitterID
```

Comment:
This is unfortunately on my cleanup list... tricking the `didSet` pattern isn't great.